hoot-core-test/src/test/cpp/hoot/core/conflate/highway/HighwayMatchTest.cpp

```diff
@@ -108,17 +108,17 @@ class HighwayMatchTest : public HootTestFixture
     std::shared_ptr<MaximalSublineStringMatcher> sublineMatcher(new MaximalSublineStringMatcher());
     sublineMatcher->setMinSplitSize(5.0);
     sublineMatcher->setMaxRelevantAngle(toRadians(60.0));
-    HighwayMatch match12(classifier, sublineMatcher, map, w1->getElementId(), w2->getElementId(),
-      mt);
+    MatchPtr match12(new HighwayMatch(classifier, sublineMatcher, map, w1->getElementId(), w2->getElementId(),
```

Comment:
Converting to `MatchPtr` (`shared_ptr<Match>`) because it is used everywhere else so that `Match` objects aren't leaked.